cogs/urbandict_utils/urbandictpages.py

```diff
@@ -0,0 +1,42 @@
+import discord
+from discord.ext import commands
+import re
+
+from cogs.utils.paginator import Pages
```

Comment:
We need to wait until paginator is implemented and merged into master from #21 as this will not work otherwise

Reply:
is it legit tho?